build/tests/Dockerfile

```diff
@@ -13,9 +13,9 @@ USER root
 # |
 # | Libraries used in the build process of this image.
 # |
-RUN apt-get update
-RUN apt-get install -y build-essential
-RUN apt-get install manpages-dev
+RUN apt-get update \
+  && apt-get install -y build-essential \
+  && apt-get install manpages-dev
```

Comment:
This is just something my Dockerfile linter recommended. It's generally better to avoid multiple run statements in a row as they create separate layers in the docker image unnecessarily. Not a big deal either way though.